resources/logo.svg

```diff
@@ -1,151 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why deleted?

Reply:
Because I am not sure it was used anywhere (maybe in an old attempt/version of the docs web pages?).
There was also an `index.html` file which did nothing.
I don't see the svg file used anywhere, all the other repos are using a png for the logo.
But maybe we want to move to using svgs instead?

Reply:
> I don't see the svg file used anywhere, all the other repos are using a png for the logo.

The `png` is always created from the `svg` (manually), so the `svg` is not "unused".